plugins/plugin-chart-echarts/src/Funnel/controlPanel.tsx

```diff
@@ -0,0 +1,204 @@
+        [legendMarginControl],
+        // eslint-disable-next-line react/jsx-key
+        [<h1 className="section-header">{t('Labels')}</h1>],
+        [
+          {
+            name: 'label_type',
+            config: {
+              type: 'SelectControl',
+              label: t('Label Type'),
+              default: labelType,
+              renderTrigger: true,
+              choices: [
+                ['key', 'Category Name'],
+                ['value', 'Value'],
+                ['percent', 'Percentage'],
+                ['key_value', 'Category and Value'],
+                ['key_percent', 'Category and Percentage'],
+                ['key_value_percent', 'Category, Value and Percentage'],
+              ] as [EchartsFunnelLabelTypeType, string][],
+              description: t('What should be shown on the label?'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'number_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Number format'),
+              renderTrigger: true,
+              default: numberFormat,
+              choices: D3_FORMAT_OPTIONS,
+              description: `${t('D3 format syntax: https://github.com/d3/d3-format')} ${t(
+                'Only applies when "Label Type" is set to show values.',
+              )}`,
+            },
+          },
+        ],
+        [
+          {
+            name: 'show_labels',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show Labels'),
+              renderTrigger: true,
+              default: showLabels,
+              description: t('Whether to display the labels.'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'label_line',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Label Line'),
+              default: labelLine,
+              renderTrigger: true,
+              description: t('Draw line from Funnel to label when labels outside?'),
+            },
+          },
+          {
+            name: 'min',
+            config: {
+              type: 'SliderControl',
+              label: t('min'),
+              default: min,
+              renderTrigger: true,
+              description: t('The minimum value of the specified data.'),
+            },
+          },
+          {
+            name: 'max',
+            config: {
+              type: 'SliderControl',
+              label: t('min'),
+              default: max,
+              renderTrigger: true,
+              description: t('The maximum value of the specified data.'),
+            },
+          },
+          {
+            name: 'gap',
+            config: {
+              type: 'SliderControl',
+              label: t('gap'),
+              default: gap,
+              renderTrigger: true,
+              description: t('Data graph spacing.'),
+            },
+          },
```

Comment:
The slider needs a minimum and maximum to function properly (now it's only returning `undefined` which breaks the chart). I'd almost recommend removing this control and just default to `0` for gap (we can later add this customization option if it's needed, but I doubt it will be needed).